resources/deployments/controller/controller_deployments_external_test.go

```diff
@@ -44,6 +46,16 @@ func makeDeviceAuthHeader(claim string) string {
 		base64.StdEncoding.EncodeToString([]byte(claim)))
 }
 
+func makeApi(router rest.App) *rest.Api {
+	api := rest.NewApi()
+	api.Use(
+		&requestlog.RequestLogMiddleware{},
```

Comment:
I think we can take that as an enhancement and fix it later. How does that sound?